features/fields.feature

```diff
@@ -268,23 +310,17 @@ Functionaliteit: Fields
       | type                | RaadpleegMetBurgerservicenummer |
       | burgerservicenummer | 999992077                       |
       | fields              | burgerservicenummer             |
-      Dan heeft de persoon met burgerservicenummer '999992077' alleen de volgende gegevens
+      Dan heeft de response een persoon met alleen de volgende gegevens

```

Comment:
Ik zie nu dat in dit scenario twee keer "Gegeven" staat (300 en 303). De tweede keer moet "en" worden.

Reply:
aangepast